### ISSUE TITLE
Warnings section of Vehicle Designer window cuts off longer warnings.

### ISSUE THREAD

**Issue by P-v-K** [ekolis/FrEee#182]
**Describe the bug**
The Warnings section of Vehicle Designer window cuts off longer warnings, making them unreadable.

**To Reproduce**
1. Create a new design, hull type Small Transport.
2. Look at the Warnings section. Notice for example the 50% warning can't be read enough to see 50% of WHAT?
3. Try to stretch or scroll the window or the section - no apparent way to do that. Can select the warning but can't Copy the text, either.

**Expected behavior**
Text should wrap, or the UI should provide SOME way to possibly read all of the content in every view.

**Actual behavior**
As above.
![warningcrop](https://user-images.githubusercontent.com/30736702/107130574-ab5dfc80-6883-11eb-8365-61890a5ab217.png)

**Comment by ekolis** [comment]
I swear someone fixed this already...

**Comment by ekolis** [comment]
Yep, this has been fixed; the warnings now wrap to the next line.
